gtd-timer/Services/SignUpService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Linq;
+
+using gtdtimer.Exceptions;
+using gtdtimer.Extensions;
+using gtdtimer.Timer.DAL.Entities;
+using gtdtimer.Timer.DAL.UnitOfWork;
+using gtdtimer.Timer.DTO;
+
+namespace gtdtimer.Services
+{
+    public class SignUpService : BaseService, ISignUpService
+    {
+        public SignUpService(IUnitOfWork unitOfWork) : base(unitOfWork)
+        {
+        }
+
+        public void Add(UserDTO model)
+        {
+            if (IsExist(model))
+            {
+                throw new UserExistException();
+            }
+
+            User user = model.ToUser();
+
+            unitOfWork.Users.Create(user);
+            unitOfWork.Save();
+        }
+
+        public User GetUser(int id)
+        {
+            var user = unitOfWork.Users.GetByID(id);
+
+            return user;
+        }
+
+        private bool IsExist(UserDTO model)
+        {
+            var userExist = unitOfWork.Users.GetAll().Any(u => u.Email == model.Email);
```

Comment:
this should be change to use UserManager once we have it.

Reply:
It will be change after implement UserManager

Reply:
again - ticket to not forget about it

Reply:
Issue #30 created